app/src/main/java/io/neurolab/NeuroSettingsFragment.java

```diff
@@ -17,18 +24,12 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
         PreferenceScreen preferenceScreen = getPreferenceScreen();
         int count = preferenceScreen.getPreferenceCount();
 
-        for (int i = 0; i < count; i++) {
-            Preference pref = preferenceScreen.getPreference(i);
-            String value = sharedPreferences.getString(pref.getKey(), "");
-            pref.setSummary(value);
-        }
+
     }
 
     @Override
     public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, String key) {
-        Preference preference = findPreference(key);
-        String value = sharedPreferences.getString(preference.getKey(), "");
```

Comment:
Why adding and removing in different PRs?

Reply:
Sir because it later implied that there was literally no use of that there.